dev/com.ibm.ws.concurrent.mp_fat_jakarta/build.gradle

```diff
@@ -1,17 +1,15 @@
 /*******************************************************************************
- * Copyright (c) 2021, 2022 IBM Corporation and others.
+ * Copyright (c) 2021, 2023 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
  * 
  * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
 addRequiredLibraries.dependsOn addDerby
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
This seems odd for a bucket that is already written for jakarta only.

Reply:
When using JakartaEE9Action or JakartaEE10Action, the transformer code still runs even if there is nothing to transform. No doubt that could be improved but it is a separate issue.